target/linux/ath79/image/generic.mk

```diff
@@ -418,6 +439,21 @@ define Device/avm_fritzdvbc
 endef
 TARGET_DEVICES += avm_fritzdvbc
 
+define Device/avm_fritzx490-wasp
+  SOC := qca9558
+  DEVICE_VENDOR := AVM
+  DEVICE_MODEL := FRITZ!Box 3490/5490/7490 WASP (Wireless Assist)
+  KERNEL := kernel-bin | append-dtb | relocate-kernel | lzma-no-dict | \
+      avm-wasp-loader | pad-to 4096 | pad-extra 208 | avm-wasp-checksum x490
+  LOADER_TYPE := bin
+  KERNEL_INITRAMFS := $$(KERNEL)
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca988x-ct-full-htt \
+	kmod-owl-loader wasp_downloader wpad -wpad-basic-wolfssl \
```

Comment:
This is strange. Neither the QCA9558 SOC's wmac nor ath10k need the owl-loader.
(The owl-loader is there for AR92XX and older PCI(e) cards. Anything AR9300 and above don't need it)

Reply:
Thanks for bringing that up, I will change it and test.